xbrowse_server/templates/family/family_home.html

```diff
@@ -145,12 +146,83 @@ <h4 class="section-header">Individuals</h4>
                         <td>{% if indiv.maternal_id %}{{ indiv.maternal_id }}{% else %}-{% endif %}</td>
                         <td>{% if indiv.paternal_id %}{{ indiv.paternal_id }}{% else %}-{% endif %}</td>
                         <td>{% for ipheno in indiv.get_phenotypes %}{{ ipheno.phenotype.name }}{% empty %}-{% endfor %}</td>
+                        <td><button id={{ indiv }}_{{ project.project_id }} type="button" class="btn btn-info btn-sm phenotipsEditModalBtn">Edit in Phenotips</button></td>
                     </tr>
+                    
+                    
                     {% endfor %}
                     </tbody>
                 </table>
             </div>
             
+
+
+        <!-- Phenotips integration -->
+		<div id="phenotipsModal" class="modal" role="dialog">
+			<div class="modal-dialog">
+				<div class="modal-content">
+					<div class="modal-header">
+						<button type="button" class="close" data-dismiss="modal">&times;</button>
```

Comment:
cool.. we should add this X to other xBrowse dialogs (like gene details in variant search results)